src/backend/distributed/planner/relation_restriction_equivalence.c

```diff
@@ -590,7 +613,7 @@ GenerateAllAttributeEquivalences(PlannerRestrictionContext *plannerRestrictionCo
 		plannerRestrictionContext->joinRestrictionContext;
 
 	/* reset the equivalence id counter per call to prevent overflows */
-	attributeEquivalenceId = 1;
```

Comment:
Passing down a struct etc seems good but then it seems to be adding more complexity, do you think we should just make the first letter capital so that it is at least known to be a global variable compared to the previous version? @marcocitus

Reply:
makes sense